cli/src/api-key-success.html

```diff
@@ -1,254 +1,173 @@
+          fill="#ffffff" />
+        <path
+          d="M520.949 241.956c-.044 37.772-14.921 73.979-41.358 100.656l-.003-.003-93.787 94.639c-3.25 3.279-8.529 3.289-11.575-.186-22.801-26.019-35.472-59.67-35.432-94.632.044-37.772 14.921-73.98 41.358-100.657l.003.003 85.686-86.464c7.723-7.794 20.36-7.849 26.91.975 18.231 24.563 28.234 54.599 28.198 85.669z"
+          fill="url(#paint0_linear_48_446)" />
+        <path
+          d="M185.414 343.189c-26.437-26.677-41.314-62.885-41.358-100.657-.035-31.07 9.968-61.105 28.198-85.668 6.55-8.825 19.187-8.769 26.911-.976l85.686 86.465.002-.002c26.438 26.677 41.315 62.884 41.358 100.656.041 34.962-12.63 68.614-35.432 94.632-3.045 3.475-8.325 3.466-11.575.187l-93.787-94.639-.003.002z"
+          fill="url(#paint1_linear_48_446)" />
+        <path
+          d="M476.062 512.532c-6.89 8.558-19.493 8.483-27.2.67l-52.926-53.657c-3.247-3.293-3.241-8.629.014-11.913l87.904-88.703c3.25-3.279 8.55-3.298 11.416.329 15.257 19.305 21.256 45.985 16.72 75.358-4.151 26.887-16.848 54.218-35.928 77.916z"
+          fill="url(#paint2_linear_48_446)" />
+        <path
+          d="M216.144 513.778c-7.706 7.813-20.309 7.888-27.199-.67-19.08-23.697-31.778-51.029-35.929-77.915-4.535-29.374 1.464-56.054 16.72-75.359 2.867-3.627 8.167-3.608 11.417-.329l87.904 88.703c3.255 3.285 3.261 8.621.013 11.913l-52.926 53.657z"
+          fill="url(#paint3_linear_48_446)" />
         <defs>
-          <clipPath id="clip0_4_2085">
-            <rect width="334" height="234" fill="white" />
-          </clipPath>
+          <linearGradient id="paint0_linear_48_446" x1="332.503" y1="149.563" x2="332.503" y2="443.522"
+            gradientUnits="userSpaceOnUse">
+            <stop stop-color="#9F87FF" />
+            <stop offset="1" stop-color="#8566FF" />
+          </linearGradient>
+          <linearGradient id="paint1_linear_48_446" x1="332.503" y1="149.563" x2="332.503" y2="443.522"
+            gradientUnits="userSpaceOnUse">
+            <stop stop-color="#9F87FF" />
+            <stop offset="1" stop-color="#8566FF" />
+          </linearGradient>
+          <linearGradient id="paint2_linear_48_446" x1="332.503" y1="353.019" x2="332.503" y2="519.292"
+            gradientUnits="userSpaceOnUse">
+            <stop stop-color="#DE99F6" />
+            <stop offset="1" stop-color="#D669FC" />
+          </linearGradient>
+          <linearGradient id="paint3_linear_48_446" x1="332.503" y1="353.019" x2="332.503" y2="519.292"
+            gradientUnits="userSpaceOnUse">
+            <stop stop-color="#DE99F6" />
+            <stop offset="1" stop-color="#D669FC" />
+          </linearGradient>
         </defs>
       </svg>
     </div>
+    <h1>Installation of the CLI is now complete</h1>
+
+    <p>
+      The Xata CLI is now linked to your account. An API key was added to
+      <code>~/.config/xata/credentials</code>.
+    </p>
```

Comment:
I'm not sure how strictly we view this, but `~/` may not be the home directory depending on the operating system of the user given how we calculate it:
https://github.com/xataio/client-ts/blob/22e284bcbc9398c2e76d29ecc79a335bc1ff2f66/cli/src/credentials.ts#L18


Would it be worth considering something like this? 

```suggestion
    <p>
      The Xata CLI is now linked to your account. An API key was saved to your home directory as
      <code>.config/xata/credentials</code>.
    </p>
```

Reply:
This was brought up in https://github.com/xataio/frontend-next/pull/2697 which @richardgill wanted to address in a separate PR

However you all want to solve for it. Feel free to apply.